### PR TITLE
Custom links plugin

### PR DESCRIPTION
**Pull request by Hello-Flotiq**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -124,6 +124,7 @@ nav:
       - Overview: panel/webhooks/index.md
       - Examples: panel/webhooks/examples.md
     - Plugins:
+      - 'Custom Links': panel/Plugins/Custom-links.md
```
Comment (andrzejwp) [inline mkdocs.yml:127]: This was missing in the original version of the MR. Without this - the page will not appear in the main nav or the sitemaps.

File: docs/panel/Plugins/Custom-links.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* Displayed Name Template - Any name that will be displayed on the link button.
+
+* Content Type Definition - Select the content type to display the button only for the specified content type. If the content type is not selected, the button will be shown when editing each content object.
+
+You're done. Now you will see `Link to current post` in Content Objects forms, the button  `https://my-blog/post/first-post`.
+
+![Netlify build button in Flotiq editor](images/plugins-custom-links-3.png){: .center .width50 .border}
+
+
+## Examples
```
Comment (andrzejwp) [inline docs/panel/Plugins/Custom-links.md:32]: Adding Examples section to better demonstrate how this plugin can be used

File: docs/panel/Plugins/Custom-links.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+* URL Template - It's a place to enter the link template, e.g. `https://my-blog/post/{slug}`, where `slug` is the name of the content field of the type selected below. Instead of `{slug}`, you can use any field of a given content type, it is also possible to use nesting, e.g. `{internal.createdAt}`. However, the use of list type fields is not supported.
+
+* Displayed Name Template - Any name that will be displayed on the link button.
+
+* Content Type Definition - Select the content type to display the button only for the specified content type. If the content type is not selected, the button will be shown when editing each content object.
+
+You're done. Now you will see `Link to current post` in Content Objects forms, the button  `https://my-blog/post/first-post`.
+
+![Netlify build button in Flotiq editor](images/plugins-custom-links-3.png){: .center .width50 .border}
```
Comment (andrzejwp) [inline docs/panel/Plugins/Custom-links.md:29]: Please fix the caption of this image. It's not related to netlify.

File: docs/panel/Plugins/Custom-links.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+![Flotiq plugins](images/profile-plugins.png){: .center .width25 .border}
+
+On the next screen, click on the `Add to your plugins` button next to `Custom Links`.
+
+![Adding Netlify Build plugin to Flotiq](images/plugins-custom-links-1.png){: .center .width50 .border}
+
+Next - fill in the details and click `Save changes` to finish your plugin setup.
+
+![Setting up Netlify Build in Flotiq](images/plugins-custom-links-2.png){: .center .width50 .border}
```
Comment (andrzejwp) [inline docs/panel/Plugins/Custom-links.md:19]: Please fix the caption of this image. It's not related to netlify.

**Review by andrzejwp — CHANGES_REQUESTED** [review]

**Comment by andrzejwp** [comment]
It would be nice to see a short description of changes included in this MR as well.

**Comment by andrzejwp** [comment]
This PR refers to #154

**Comment by trzcina** [comment]
I've checked the change requests and updates; merging

**Pull request merged**